rc/windowing/zellij.kak

```diff
@@ -0,0 +1,74 @@
+# https://zellij.dev/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+provide-module zellij %{
+
+# ensure we're running under zellij
+evaluate-commands %sh{
+    [ -z "${kak_opt_windowing_modules}" ] || [ -n "$ZELLIJ" ] || echo 'fail zellij not detected'
+}
+
+define-command -hidden -params 2.. zellij %{
+    evaluate-commands %sh{
+       zellij "$@" < /dev/null > /dev/null 2>&1 &
+    }
+}
+define-command -hidden -params 1.. zellij-terminal-impl %{ evaluate-commands %sh{
+   arg_count=0
+   i=1
+   while [ $i -lt $# ]; do
+       a=${@:$i:1}
+       if [ "${a:0:1}" = "-" ]; then
+           if [ "${a:1:1}" = "d" -o "${a:1:1}" = "n" ]; then
+               ((i+=2))
+           else
+               ((i+=1))
+           fi
+           arg_count=$((i-1))
+       else
+           arg_count=$((i-1))
+           i=$#
+       fi
+   done
+   echo zellij run ${@:1:$arg_count} -- $SHELL -i -c "'"${@:$((1+$arg_count))}"'"
+}}
+complete-command zellij-terminal-impl shell
+
+define-command zellij-terminal-vertical -params 1.. -docstring '
+zellij-terminal-vertical <program> [<arguments>]: create a new terminal as a zellij pane
+The current pane is split into two, top and bottom
+The program passed as argument will be executed in the new terminal' \
+%{
+    zellij-terminal-impl -d down %arg{@}
+}
+complete-command zellij-terminal-vertical shell
+
+define-command zellij-terminal-horizontal -params 1.. -docstring '
+zellij-terminal-horizontal <program> [<arguments>]: create a new terminal as a zellij pane
+The current pane is split into two, left and right
+The program passed as argument will be executed in the new terminal' \
+%{
+    zellij-terminal-impl -d right %arg{@}
+}
+complete-command zellij-terminal-horizontal shell
+
+define-command zellij-terminal-window-edit -params .. -docstring '
+zellij-terminal-window-edit <file>: create a new window to edit given file' \
```

Comment:
This seems equivalent to `new edit <file>` I dont think we need to add a new alias just for that use case.

Reply:
Good point, fixed now, thanks!